k8s-oci.tf

```diff
@@ -471,9 +471,14 @@ module "k8smaster-public-lb" {
   shape                     = "${var.k8sMasterLBShape}"
 }
 
+module "nginx-lb" {
+  source = "./network/loadbalancers/nginx"
+  hosts  = "${concat(module.instances-k8smaster-ad1.private_ips,module.instances-k8smaster-ad2.private_ips, module.instances-k8smaster-ad3.private_ips)}"
+}
+
 module "kubeconfig" {
   source                     = "./kubernetes/kubeconfig"
   api_server_private_key_pem = "${module.k8s-tls.api_server_private_key_pem}"
   api_server_cert_pem        = "${module.k8s-tls.api_server_cert_pem}"
-  k8s_master                 = "https://${module.k8smaster-public-lb.ip_addresses[0]}:443"
+  k8s_master                 = "{local.master_address}"
```

Comment:
Typo... should be `"${local.master_address}"`

Reply:
It can only be configured to the address of master, since it is the address of k8s api-server which exits only in master.  We can not set it to `worker_public_ip`.

When master_lb_enabled = false, api-server can't be accessed outside the cluster, so we have to run a nginx locally in NAT hosts which reverse proxy to the private addresses of masters.